### PR TITLE
Address QA items for styling

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/PURCHASE-94

These are small items @briansw and I discussed earlier today:

  * Removed the "Artsy's" from the Condition of Sale
  * The margin between the CoS and the PLACE BID button is now 30px
  * Changed the font size of the title to 22px
  * Set maxWidth to the bid result screen to 

screen shot 2018-04-25 at 6 18 39 pm

cc @briansw 

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx">BidResult-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx#L12'>renders winning screen properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -51,15 +51,17 @@
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          fontSize={4}
+         mb={4}
          style={
            Array [
              Object {
                "fontFamily": "AGaramondPro-Semibold",
                "fontSize": 18,
+               "marginBottom": 20,
              },
              undefined,
            ]
          }
        >
@@ -69,18 +71,16 @@
          accessible={true}
          allowFontScaling={true}
          color="black60"
          ellipsizeMode="tail"
          fontSize={1}
-         mt={3}
          style={
            Array [
              Object {
                "color": "#666",
                "fontFamily": "Unica77LL-Medium",
                "fontSize": 12,
-               "marginTop": 10,
              },
              undefined,
            ]
          }
        >
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx:12:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>



<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx#L21'>renders not highest bid screen properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -59,15 +59,17 @@
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        fontSize={4}
+       mb={4}
        style={
          Array [
            Object {
              "fontFamily": "AGaramondPro-Semibold",
              "fontSize": 18,
+             "marginBottom": 20,
            },
            undefined,
          ]
        }
      >
@@ -77,21 +79,20 @@
        accessible={true}
        allowFontScaling={true}
        color="black60"
        ellipsizeMode="tail"
        fontSize={3}
-       m={3}
+       maxWidth={280}
+       mb={5}
        style={
          Array [
            Object {
              "color": "#666",
              "fontFamily": "AGaramondPro-Regular",
              "fontSize": 16,
-             "marginBottom": 10,
-             "marginLeft": 10,
-             "marginRight": 10,
-             "marginTop": 10,
+             "marginBottom": 30,
+             "maxWidth": 280,
              "textAlign": "center",
            },
            undefined,
          ]
        }
@@ -103,21 +104,18 @@
        accessible={true}
        allowFontScaling={true}
        color="black60"
        ellipsizeMode="tail"
        fontSize={3}
-       m={3}
+       mb={5}
        style={
          Array [
            Object {
              "color": "#666",
              "fontFamily": "AGaramondPro-Regular",
              "fontSize": 16,
-             "marginBottom": 10,
-             "marginLeft": 10,
-             "marginRight": 10,
-             "marginTop": 10,
+             "marginBottom": 30,
              "textAlign": "center",
            },
            undefined,
          ]
        }
@@ -129,18 +127,16 @@
        accessible={true}
        allowFontScaling={true}
        color="black60"
        ellipsizeMode="tail"
        fontSize={1}
-       mt={3}
        style={
          Array [
            Object {
              "color": "#666",
              "fontFamily": "Unica77LL-Medium",
              "fontSize": 12,
-             "marginTop": 10,
            },
            undefined,
          ]
        }
      >
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx:21:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx">ConfirmBid-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx#L19'>renders properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -20,21 +20,21 @@
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
-     fontSize={4}
-     m={5}
+     fontSize={5}
+     m={4}
      style={
        Array [
          Object {
            "fontFamily": "AGaramondPro-Semibold",
-           "fontSize": 18,
-           "marginBottom": 30,
-           "marginLeft": 30,
-           "marginRight": 30,
-           "marginTop": 30,
+           "fontSize": 22,
+           "marginBottom": 20,
+           "marginLeft": 20,
+           "marginRight": 20,
+           "marginTop": 20,
            "textAlign": "center",
          },
          undefined,
        ]
      }
@@ -251,24 +251,26 @@
        accessible={true}
        allowFontScaling={true}
        color="black60"
        ellipsizeMode="tail"
        fontSize={2}
+       mb={3}
        style={
          Array [
            Object {
              "color": "#666",
              "fontFamily": "AGaramondPro-Regular",
              "fontSize": 14,
+             "marginBottom": 10,
              "textAlign": "center",
            },
            undefined,
          ]
        }
        textAlign="center"
      >
-       You agree to Artsy's 
+       You agree to 
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
@@ -278,12 +280,13 @@
              },
              undefined,
            ]
          }
        >
-         Condition of Sale.
+         Condition of Sale
        </Text>
+       .
      </Text>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx:19:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Containers/__tests__/BidFlow-tests.tsx">BidFlow-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Containers/__tests__/BidFlow-tests.tsx#L34'>renders properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -70,41 +70,44 @@
      >
        <View
          flex={1}
          flexDirection="column"
          justifyContent="space-between"
-         m={3}
+         m={0}
          style={
            Array [
              Object {
                "flexBasis": 0,
                "flexGrow": 1,
                "flexShrink": 1,
-               "marginBottom": 10,
-               "marginLeft": 10,
-               "marginRight": 10,
-               "marginTop": 10,
+               "marginBottom": 0,
+               "marginLeft": 0,
+               "marginRight": 0,
+               "marginTop": 0,
              },
              undefined,
            ]
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
-           fontSize={4}
+           fontSize={5}
+           m={4}
            style={
              Array [
                Object {
                  "fontFamily": "AGaramondPro-Semibold",
-                 "fontSize": 18,
+                 "fontSize": 22,
+                 "marginBottom": 20,
+                 "marginLeft": 20,
+                 "marginRight": 20,
+                 "marginTop": 20,
                  "textAlign": "center",
                },
-               Object {
-                 "margin": 10,
-               },
+               undefined,
              ]
            }
            textAlign="center"
          >
            Your max bid
@@ -184,11 +187,11 @@
                "backgroundColor": "rgba(0, 0, 0, 1)",
                "borderColor": "rgba(0, 0, 0, 1)",
                "borderWidth": 1,
                "height": 46,
                "justifyContent": "center",
-               "margin": 10,
+               "margin": 20,
              }
            }
            testID={undefined}
          >
            <Text
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Containers/__tests__/BidFlow-tests.tsx:34:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/SelectMaxBid-tests.tsx">SelectMaxBid-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Screens/__tests__/SelectMaxBid-tests.tsx#L34'>renders properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -1,40 +1,43 @@
  <View
    flex={1}
    flexDirection="column"
    justifyContent="space-between"
-   m={3}
+   m={0}
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
-         "marginBottom": 10,
-         "marginLeft": 10,
-         "marginRight": 10,
-         "marginTop": 10,
+         "marginBottom": 0,
+         "marginLeft": 0,
+         "marginRight": 0,
+         "marginTop": 0,
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
-     fontSize={4}
+     fontSize={5}
+     m={4}
      style={
        Array [
          Object {
            "fontFamily": "AGaramondPro-Semibold",
-           "fontSize": 18,
+           "fontSize": 22,
+           "marginBottom": 20,
+           "marginLeft": 20,
+           "marginRight": 20,
+           "marginTop": 20,
            "textAlign": "center",
          },
-         Object {
-           "margin": 10,
-         },
+         undefined,
        ]
      }
      textAlign="center"
    >
      Your max bid
@@ -114,11 +117,11 @@
          "backgroundColor": "rgba(0, 0, 0, 1)",
          "borderColor": "rgba(0, 0, 0, 1)",
          "borderWidth": 1,
          "height": 46,
          "justifyContent": "center",
-         "margin": 10,
+         "margin": 20,
        }
      }
      testID={undefined}
    >
      <Text
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Bidding/Screens/__tests__/SelectMaxBid-tests.tsx:34:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Components/__tests__/Title-tests.tsx">Title-tests.tsx</a>


<li>
<a href='https://github.com/artsy/emission/blob/ui-tweaks/src/lib/Components/Bidding/Components/__tests__/Title-tests.tsx#L9'>renders properly</a>
<br/>
Snapshot has changed

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(value).toMatchSnapshot()

Received value does not match stored snapshot 1.

- Snapshot
+ Received

@@ -1,15 +1,20 @@
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
-   fontSize={4}
+   fontSize={5}
+   m={4}
    style={
      Array [
        Object {
          "fontFamily": "AGaramondPro-Semibold",
-         "fontSize": 24,
+         "fontSize": 32,
+         "marginBottom": 32,
+         "marginLeft": 32,
+         "marginRight": 32,
+         "marginTop": 32,
          "textAlign": "center",
        },
        undefined,
      ]
    }
    at Object.<anonymous> (/home/travis/build/artsy/emission/src/lib/Components/Bidding/Components/__tests__/Title-tests.tsx:9:14)
    at Object.asyncFn (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
    at resolve (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
    at Promise (<anonymous>)
    at mapper (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:39:19)
    at promise.then (/home/travis/build/artsy/emission/node_modules/jest-jasmine2/build/queue_runner.js:73:82)
    at <anonymous>
</code></pre>
</details>
</li>
<br/>


  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Bidding/Elements/__tests__/Typography-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Bidding/Components/Title.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react"
 
-import { SerifSemibold18 } from "../Elements/Typography"
+import { SerifSemibold22 } from "../Elements/Typography"
 
-export const Title = props => <SerifSemibold18 textAlign="center" {...props} />
+export const Title = props => <SerifSemibold22 m={4} textAlign="center" {...props} />
```
Comment (sweir27) [inline src/lib/Components/Bidding/Components/Title.tsx]: 😄 nice to see the abstraction already paying off!

**Comment by yuki24** [comment]
The CI build doesn't look like a legit failure. I don't think this PR deserves a CHANGELOG entry, so I'll just go ahead and merge this PR.

**Comment by yuki24** [comment]
Now that we have the automated deploy or Emission fixed, I think I should have added a new CHANGELOG entry.

**Pull request merged**